tests/test_filters.py

```diff
@@ -747,6 +747,7 @@ def test_empty_choices(self):
 
     def test_default_field_with_queryset(self):
         qs = mock.NonCallableMock(spec=[])
+        qs.all = mock.Mock(return_value=qs)  # For django 2.1+
```

Comment:
Out of curiosity, why is this necessary/what does this fix?

Reply:
https://github.com/django/django/pull/9725/files